### DISCUSSION TITLE
安装报错 Python版本3.11

### DISCUSSION THREAD

**Discussion by Baili-BL** [shepherdpp/qteasy#128] [Show and tell]
![image](https://github.com/shepherdpp/qteasy/assets/35422348/d9ef3d8e-51ce-4882-8dda-f45350c0e148)

可能我的Python环境是3.11版本的，按照会报错，大佬看下这个问题怎么解决。我尝试了卸载再安装没有成功。

**Comment by shepherdpp** [comment]
错误信息说明`numba`与高版本的python不兼容。Qteasy还没有在高版本的python环境中充分测试过，因此目前限定用于python3.7到python3.10。
建议您创建一个新的虚拟环境并安装python3.8

建议使用anaconda:
```
conda create -n qteasy-env python=3.8
conda activate qteasy-env
```

**Reply by shepherdpp** [reply]
在后续的升级版本中，将重点测试高版本python环境，修复兼容性问题

**Reply by Baili-BL** [reply]
好的，谢谢

**Reply by shepherdpp** [reply]
现在qteasy: v1.1.7 可以在python3.9～3.12的版本上安装并导入了

**Comment by shepherdpp** [comment]
pip install the latest update of qteasy (v1.1.7), which has been tested for installation and importing on higher versions of python (from 3.9 up to 3.12)

**Reply by Baili-BL** [reply]
感谢大佬的速度。
